fedmsg_meta_fedora_infrastructure/buildsys.py

```diff
@@ -274,5 +281,7 @@ def objects(self, msg, **config):
                 'tags',
                 msg['msg'].get('tag', 'unknown'),
             ])])
-        else:
-            raise NotImplementedError("%r" % msg)
+        elif 'buildsys.rpm.sign' in msg['topic']:
+            return set(['signatures/' + msg['msg']['info']['name']])
+
```

Comment:
Right, so the return set is there, but no else. I guess since the rest is also return's, it's okay.